static/js/shared/types.ts

```diff
@@ -27,6 +27,12 @@ export interface NamedNode {
 
 export type NamedPlace = NamedNode;
 
+// Place with name and its type.
```

Comment:
Nit: please use similar style to the other interfaces in this file (and add an empty line after)

Reply:
done